modules/nw-how-nw-iface-selected.adoc

```diff
@@ -0,0 +1,78 @@
+// Applies to 4.6 and newer.
+
+:ign-config-version: 3.1.0
+ifeval::[{product-version} > 4.6]
+:ign-config-version: 3.2.0
+endif::[]
+
+[id="nw-how-nw-iface-selected_{context}"]
+= How the network interface is selected
+
+For deployments on bare metal or with virtual machines that have more than one network interface controller (NIC), the NIC that {product-title} uses for communication with the Kubernetes API server is determined by a script that runs as a systemd unit.
+The `nodeip-configuration.service` systemd unit runs when the node boots.
```

Comment:
Can you combine these two sentences without it being too long? Something like:
_...is determined by the `nodeip-configuration.service` script that is executed by the systemd unit when the node boots."_

Reply:
I was forcing it into two sentences.  It prolly should be four sentences, but is slightly less bumpy as a single sentence.